src/ast/bpforc/bpforcv2.cpp

```diff
@@ -1,5 +1,23 @@
 // Included by bpforc.cpp
 
+#if LLVM_VERSION_MAJOR >= 13
+BpfOrc::BpfOrc(TargetMachine *TM,
```

Comment:
Would it make sense to make the ES a member in <13 too? That way we can drop this if and only keep the #if in the .h

Reply:
I have addressed this in revised pull. Please review again @fbs